crates/terminal/src/mappings/keys.rs

```diff
@@ -54,6 +54,7 @@ pub fn to_esc_str(keystroke: &Keystroke, mode: &TermMode, alt_is_meta: bool) ->
         ("backspace", AlacModifiers::None) => Some("\x7f".to_string()),
         //Interesting escape codes
         ("tab", AlacModifiers::Shift) => Some("\x1b[Z".to_string()),
+        ("backspace", AlacModifiers::Ctrl) => Some("\x08".to_string()),
```

Comment:
I used the Control-H key here since that's what Konsole and Gnome Terminal send when doing Ctrl+Backspace. VSCode sends `\x1b\x7f` like when doing Alt+Backspace.

Reply:
I think I'd prefer if we didn't make this change, everything else being equal.

Reply:
This seems to be the expected behaviour: https://github.com/alacritty/alacritty/issues/7100#issuecomment-1645035896
Asides, if the keymap overwrites the keybinding (like in macOS, where I did not change it) it takes precedence over the code here.

Reply:
Oh excellent. We should stick to alacritty then. Thanks for clarifying :)